packages/fxa-auth-server/config/index.ts

```diff
@@ -780,6 +780,26 @@ const conf = convict({
         env: 'PAYPAL_NVP_SIGNATURE',
       },
     },
+    appStore: {
+      credentials: {
+        doc: 'Map of AppStore Connect credentials by app bundle ID',
+        format: Object,
+        default: {
+          'org.mozilla.ios.FirefoxVPN': {
```

Comment:
Unfortunately, `node-convict` doesn't support keys with dots in them (https://github.com/mozilla/node-convict/issues/250), so `yarn start` fails.

I've replaced the dots here with underscores, but that means we'd have make a copy of the `bundleId`s with this string replace operation to get API clients keyed off them. Not a great solution. Would you have any suggestions?

Reply:
Ben suggested using the new `getBundleId` method and key off `appName` instead, so I'll do that.